### PR TITLE
fix: Crash in MapLayer - "Contour Lines" story

### PR DESCRIPTION
**Pull request by nilscb**

### CODE REVIEW

File: typescript/packages/subsurface-viewer/src/layers/map/mapLayer.stories.tsx [hunk 1]
```diff
@@ -830,7 +830,7 @@ const ContourLinesComponent: React.FC<ContourLinesComponentProps> = (
 export const ContourLines: StoryObj<typeof ContourLinesComponent> = {
     args: {
         syncViewports: true,
-        show3d: true,
+        show3d: false,
```
Comment (t0oF-azpn) [inline typescript/packages/subsurface-viewer/src/layers/map/mapLayer.stories.tsx:833]: Why changing this ?

Comment (nilscb) [reply typescript/packages/subsurface-viewer/src/layers/map/mapLayer.stories.tsx:833]: I believe it originally was 2D and I don't see a point in having it 3D but I can change it if you will.

Comment (t0oF-azpn) [reply typescript/packages/subsurface-viewer/src/layers/map/mapLayer.stories.tsx:833]: No, it's fine.

**Review by t0oF-azpn — APPROVED** [review]

**Pull request merged**